### PR TITLE
Homepage

### PR DESCRIPTION
**Pull request by BipinPoudel1**
Here is my pull request for the Homepage section of our Project. Please review it and feel free to suggest any corrections.

### CODE REVIEW

File: pics/3821163.jpg
Comment (Madara-coder) [inline pics/3821163.jpg:1]: picture's name could have been better.

Comment (BipinPoudel1) [reply pics/3821163.jpg:1]: Oh I used the original (downloaded) names. I will take care from now on.

File: pics/favicon-32x32.3699f19f7fdff1a556c0953c12fb883f.png
Comment (Madara-coder) [inline pics/favicon-32x32.3699f19f7fdff1a556c0953c12fb883f.png:1]: The name could be given from the  (alt = "") tag of that picture.

Comment (BipinPoudel1) [reply pics/favicon-32x32.3699f19f7fdff1a556c0953c12fb883f.png:1]: Sure...

File: css/login.css
Comment (Madara-coder) [inline css/login.css:1]: why is the previous files being deleted while creating the merge request?

File: html/index.html
Comment (Madara-coder) [inline html/index.html:1]: `git pull origin main` should be done before merging your codes into the main branch.

Comment (BipinPoudel1) [reply html/index.html:1]: shall i do it again?

**Pull request closed without merging**